apps/chat-e2e/src/ui/webElements/publishedPromptPreviewModal.ts

```diff
@@ -0,0 +1,18 @@
+import { PublicationReviewControls } from '@/src/ui/selectors';
+import { promptPreviewModal } from '@/src/ui/webElements/promptPreviewModal';
+
+export class PublishedPromptPreviewModal extends promptPreviewModal {
+  public reviewContainer = this.getChildElementBySelector(
```

Comment:
you need the reference to an object of `PublicationReviewControl` class here

Reply:
done